lib/react_client.dart

```diff
@@ -643,7 +756,7 @@ class ReactDomComponentFactoryProxy extends ReactComponentFactoryProxy {
     var convertibleProps = new Map.from(props);
     convertProps(convertibleProps);
 
-    return factory(jsify(convertibleProps), children);
+    return factory(jsifyAndAllowInterop(convertibleProps), children);
```

Comment:
So that warning, is actually from the `jsify` doc comment. The only difference between `jsifyAndAllowInterop` and `jsify` is that it in addition to all the stuff `jsify` does it also wraps `allowInterop` around dart functions in the map.